src/views/Player.vue

```diff
@@ -125,7 +126,7 @@ export default {
       },
       journey: [],
       hasVideoPlayed: -1, // Three possible values: -1(don't know), 0(didn't play), 1(played)
-      sessionId: 1,
+      sessionId: 0,
```

Comment:
missed this in earlier PR. We had a mutual decision to start sessions from 0. Changing here doesn't affect the functionality. This is just for consistency